libs/s3/iam_test_lib.py

```diff
@@ -171,10 +148,8 @@ def delete_user(self, user_name: str = None) -> tuple:
             response = poll(super().delete_user, user_name)
             LOGGER.info(response)
         except (ClientError, Exception) as error:
-            LOGGER.error("Error in %s: %s",
-                         IamTestLib.delete_user.__name__,
-                         error)
-            raise CTException(err.S3_CLIENT_ERROR, error.args[0])
+            LOGGER.error("Error in %s: %s", IamTestLib.delete_user.__name__, error)
```

Comment:
Check above comments . It's a similar pattern.

Reply:
done